simple-post-view/function.py

```diff
@@ -0,0 +1,70 @@
+"""
+    :module_name: function
+    :module_summary: service to deliver high-level post information view
+    :module_author: Utsav Sampat
+"""
+
+import logging
+import json
+import boto3
+from ideadb_handler import *
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+LOG_HANDLER = logging.StreamHandler()
+LOG_HANDLER.setLevel(logging.DEBUG)
+
+LOG_FORMAT = logging.Formatter('[%(asctime)s|%(name)s|%(levelname)s] - %(message)s')
+LOG_HANDLER.setFormatter(LOG_FORMAT)
+LOGGER.addHandler(LOG_HANDLER)
+
+
+def handler(event, context):
+    """
+        Service to query high-level post details from the database
+    """
+    LOGGER.info("Start service: simple-post-view")
+    LOGGER.debug("Event info: %s", json.dumps(event, indent=4))
+
+    _input = event['body']
+    if not 'IdeaPostID' in _input: raise NotValidParameters('IdeaPostID')
+    if not 'IdeaAuthorID' in _input: raise NotValidParameters('IdeaAuthorID')
+
+    try:
+        posts = IdeaPostTable()
+        posts.load()
+        
+        LOGGER.info("Attempting to get post by IdeaPostID and IdeaAuthorID...")
+        post = posts.get_post(_input['IdeaPostID'], _input['IdeaAuthorID'])
+        
+        
+        return {
```

Comment:
Please see [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html) for the correct handler return value structure. Required or this expect integration headaches